src/standalone/Soil/Soil.jl

```diff
@@ -253,7 +253,7 @@ function EnergyHydrology(
     S_s = ClimaCore.Fields.zeros(domain.space.subsurface) .+ 1e-3,
     z_0m = LP.get_default_parameter(FT, :soil_momentum_roughness_length),
     z_0b = LP.get_default_parameter(FT, :soil_scalar_roughness_length),
-    emissivity = LP.get_default_parameter(FT, :soil_bare_soil),
+    emissivity = LP.get_default_parameter(FT, :emissivity_bare_soil),
```

Comment:
multiple typos from last PR. the good news is that as of this PR, this constructor is tested in CI